packages-proprietary/react-icons/src/index.ts

```diff
@@ -177,7 +177,7 @@ export { default as LightningFillIcon } from './LightningFill'
 export { default as LineChartDownIcon } from './LineChartDown'
 export { default as LineChartUpIcon } from './LineChartUp'
 export { default as LinkIcon } from './Link'
-export { default as LinkedinIcon } from './Linkedin'
+export { default as LinkedInIcon } from './LinkedIn'
```

Comment:
This is currently a breaking change. Shouldn't we export both spellings, with a deprecation warning for `LinkedinIcon`?

Handling the SVG in `assets` is a little trickier though.

Reply:
Alright, let’s travel the royal road. I reverted the breaking change to the name of the SVG icon. The `generate` script doesn’t overwrite the file of the React icon – probably exactly because the file system is working case-insensitively. So we can keep the deprecation and the improved name of the React components. And I’ll create a ticket to rename the asset just before we publish our next major release. Is that acceptable?